examples/plot/plot-layer/axes-layer.js

```diff
@@ -154,18 +158,18 @@ export default class AxesLayer extends Layer {
 
   updateAttributes(props) {
     super.updateAttributes(props);
-    const {attributeManager, models, numInstances} = this.state;
+    const {attributeManager, modelsByName, numInstances} = this.state;
```

Comment:
@1chandu could you comment on the models => modelsByName change? I might have missed something but I could not find refs in the code why we are destructing modelsByName instead of models from the state.

Reply:
Base Layer class uses ` for .. of` to iterate over `state.models`, it should be an array. So change `state.models` to array and created a new one `modelByName` to use with in `axes-layer`. We are doing same in `SolidPolygonLayer`.